odl/contrib/torch/operator.py

```diff
@@ -16,6 +16,12 @@
 from __future__ import division
 import numpy as np
 import torch
+import warnings
+from pkg_resources import parse_version
+
+if parse_version(torch.__version__) < parse_version('0.4'):
+    warnings.warn("This interface is designed to work"
```

Comment:
the strings will be concatenated like "workwith", you need to add a space after "work"